Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
Traceback (most recent call last):
  File "/home/ubuntu/LB2/M122/scripts/main.py", line 16, in <module>
    CONFIG = modules.config.loadconfig()
  File "/home/ubuntu/LB2/M122/scripts/modules/config.py", line 8, in loadconfig
    with open(CONFIG_PATH, "r") as stream:
FileNotFoundError: [Errno 2] No such file or directory: './config/config.yaml'
